src/tendisplus/commands/command.cpp

```diff
@@ -201,7 +201,10 @@ Command* Command::getCommand(Session* sess) {
 Expected<Command*> Command::precheck(Session* sess) {
   const auto& args = sess->getArgs();
   if (args.size() == 0) {
+    std::stringstream ss;
+    std::string fatal_message = "BUG: sess " << sess->id() << " len 0 args";
     LOG(FATAL) << "BUG: sess " << sess->id() << " len 0 args";
```

Comment:
In fact, LOG(FATAL) would coredump here.
It's not necessary to return anything.

Maybe assert here is better.